app/utils/getArchivePath.ts

```diff
@@ -83,7 +83,8 @@ const getArchivePath = (
 ) => {
   const { path } = archivePaths[fieldName];
 
-  return path.replaceAll('$id', ccbcId).replaceAll('$fileName', fileName);
+  // return path.replaceAll('$id', ccbcId).replaceAll('$fileName', fileName);
+  return path.replace(/\$id/g, ccbcId).replace(/\$fileName/g, fileName);
```

Comment:
When debugging Jest tests, got error replaceAll is not a function. Running node v16.17.1
Found related issue: https://github.com/testing-library/dom-testing-library/issues/797#issuecomment-843175877 but choose to change code rather then tweak jest.config